apis/v1alpha1/targetallocator_types.go

```diff
@@ -73,6 +73,9 @@ type TargetAllocatorSpec struct {
 	// +optional
 	// +kubebuilder:default:=relabel-config
 	FilterStrategy v1beta1.TargetAllocatorFilterStrategy `json:"filterStrategy,omitempty"`
+	// GlobalConfig configures the global configuration for Prometheus
+	// For the exact format, see https://github.com/prometheus/prometheus/blob/main/config/config.go#L409-L448
```

Comment:
I think we should link to the Prometheus documentation here, rather than the struct definition: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file